### DISCUSSION TITLE
"Error: Required parameter haId is not provided" for get_status

### DISCUSSION THREAD

**Discussion by hoeni** [alexkn/node-red-contrib-homeconnect#72] [Q&A] [answered]
Hi,

thanks for the Home Connect module!
I freshly connected my node red (after fighting for hours with safari hiding the authorization popup 🙈).

The `get_home_appliances` works fine and returns my two Home Connect devices. (see screenshot).
However, when I do a `Status/get_status` on any of both HaIds I always get an "Error: Required parameter haId is not provided" error (see screenshot).

Any hints, anyone?
![CleanShot 2022-08-09 at 19 47 17](https://user-images.githubusercontent.com/409930/183724106-59b71827-3af9-4406-b27f-d366ec7a9fb9.png)

**Comment by alexkn** [comment, marked as answer]
Are you using the latest version v0.6.5 of the node-red-contrib-homeconnect node?
There was an Issue #46 which I fixed in [v0.6.4](https://github.com/alexkn/node-red-contrib-homeconnect/releases/tag/v0.6.4)

**Reply by hoeni** [reply]
Thanks, updating to 0.6.5 fixed it!
